app/templates/views/gallery_view.html

```diff
@@ -0,0 +1,44 @@
+{% extends "/snips/base.html" %}
+{% block title %}Submissions{% endblock %}
+{% block body %}
+
+<div class="inner">
+        <h1 style="text-align:center;">Submissions</h1>
```

Comment:
Please use bootstrap for styling when possible, instead of using style tag.
Bootstrap has a text-center class that does this same action